Calendar.Console/Controllers/UpdateExistentMeetingController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Calendar.Domain.Builders;
+
+namespace Calendar.Console.Controllers
+{
+    internal class UpdateExistentMeetingController : IController
+    {
+        private readonly Context _context;
+
+        public UpdateExistentMeetingController(Context context)
+        {
+            _context = context;
+        }
+
+        public void Show()
+        {
+            WriteLine();
+            WriteLine("Enter meeting name to search:");
+        }
+
+        public IController Action()
+        {
+            string input = ReadLine();
+            if (string.IsNullOrWhiteSpace(input))
+            {
+                WriteLine("Meeting name should be not empty!");
+                return this;
+            }
+
+            if (input.Length > 20)
+            {
+                WriteLine("Meeting name length should be less than 20!");
+                return this;
+            }
+            var meeting= _context.Service.GetAllMeetings().FirstOrDefault(x=>x.Name.Equals(input));
```

Comment:
starting from here, it's better to split code to other controller
+instead of getting all meeting from service, it's better to have method `GetMeetingByName`